src/html.cr

```diff
@@ -1,11 +1,372 @@
 module HTML
-  SUBSTITUTIONS = {
-    '&'  => "&amp;",
-    '\"' => "&quot;",
-    '\'' => "&apos;",
-    '<'  => "&lt;",
-    '>'  => "&gt;",
-  }
+SUBSTITUTIONS = {
```

Comment:
Looks like indentation miss? `SUBSTITUTIONS` should be nested (by 2 spaces) inside `HTML` and all the elements in the hash also nested (by 2 spaces) from `SUBSTITUTIONS` definition.

You can use Crystal's `format` to correct that (`crystal tool format src/html.cr`)